spec/reference.md

```diff
@@ -79,8 +93,35 @@ The **continuous\_stops** field can have the following non-negative integer valu
 
 If specified as 1, a valid shape must be defined for the trip, in order to indicate the complete path of travel.
 
+###Defining Service Parameters
```

Comment:
Service parameters (travel times and booking): There may be a lot to explore here, and what I've proposed just represents a first stab. This is a start, but I fully expect major rearrangement.